lib/rules/no-get-with-default.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const types = require('../utils/types');
+
+function makeErrorMessageForGetWithDefault(property) {
+  return `Use \`get\` and \`||\` instead of \`getWithDefault\` for property \`${property}\``;
+}
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Use the || operator instead of getWithDefault for more expected behaviors',
```

Comment:
Can you add backticks around `||` and `getWithDefault`? I also noticed that this fails to render in README.md and maybe that will fix it.

Reply:
Good catch